crates/ruff_linter/src/rules/flake8_datetimez/rules/call_datetime_fromtimestamp.rs

```diff
@@ -49,14 +48,20 @@ use super::helpers;
 /// ## References
 /// - [Python documentation: Aware and Naive Objects](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)
 #[violation]
-pub struct CallDatetimeFromtimestamp;
+pub struct CallDatetimeFromtimestamp(DatetimeModuleAntipattern);
 
 impl Violation for CallDatetimeFromtimestamp {
     #[derive_message_formats]
     fn message(&self) -> String {
-        format!(
-            "The use of `datetime.datetime.fromtimestamp()` without `tz` argument is not allowed"
-        )
+        let CallDatetimeFromtimestamp(antipattern) = self;
+        match antipattern {
+            DatetimeModuleAntipattern::NoTzArgumentPassed => format!(
+                "The use of `datetime.datetime.fromtimestamp()` without `tz` argument is not allowed"
+            ),
+            DatetimeModuleAntipattern::NonePassedToTzArgument => format!(
+                "Passing `tz=None` is forbidden, as it creates a naive datetime object"
```

Comment:
```suggestion
                "Passing `tz=None` is not recommended, as it creates a naive datetime object"
```
or passing `tz=None` raise an error?